web/components/space-homepage/SpaceLandingPage.tsx

```diff
@@ -180,20 +251,28 @@ export function SpaceLandingPage() {
               items={filteredProfileListings}
               strategy={rectSortingStrategy}
             >
+
               {filteredProfileListings.map((listing, idx) => {
                 const fullName = getFullNameOfUser(listing.profile.user);
 
+              
+                
+                // this sorts the tags for each user!
```

Comment:
These comments are good for personal reference but ideally we should leave them out of the codebase because they add clutter without providing super useful information

Reply:
Agreed, will remember for future commits also.

Reply:
^^ please address!